app/handler/statuses/create.go

```diff
@@ -1,22 +1,48 @@
 package statuses
 
 import (
+	"encoding/json"
 	"fmt"
 	"net/http"
 	"yatter-backend-go/app/domain/auth"
+
+	vo "yatter-backend-go/app/domain/value-object"
 )
 
 // Request body for `POST /v1/statuses`
 type AddRequest struct {
-	Status string
+	Content string
 }
 
 // Handle request for `POST /v1/statuses`
 func (h *handler) Create(w http.ResponseWriter, r *http.Request) {
-	//ctx := r.Context()
+	var req AddRequest
+	ctx := r.Context()
 
 	account_info := auth.AccountOf(r.Context()) // 認証情報を取得する
 
-	panic(fmt.Sprintf("Must Implement Status Creation And Check Acount Info %v", account_info))
+	if err := json.NewDecoder(r.Body).Decode(&req); err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	accountID, err := vo.NewAccountID(account_info.ID.Value())
+	if err != nil {
+		http.Error(w, "Invalid account ID", http.StatusBadRequest)
+		return
+	}
```

Comment:
値オブジェクトを導入したことで、アカウントIDのバリデーションをここでまとめて行うことができるようになりましたね！ 👍 

`account_info.ID.Value()`はユーザから受け取った値ではなくDBからとってきた値なので、Internal Server Errorを返しましょう！
```suggestion
	accountID, err := vo.NewAccountID(account_info.ID.Value())
	if err != nil {
		http.Error(w, "Invalid account ID", http.StatusInternalServerError)
		return
	}
```